src/ui/layout/BrowseBar.vue

```diff
@@ -159,10 +159,16 @@ export default {
             return this.views.filter(v => v.key === this.viewKey)[0] || {};
         },
         views() {
+            let objectPath = [];
+            if (this.openmct.router.started) {
```

Comment:
@deeptailor @akhenry I want to call your attention to this.
When the router hasn't initialized yet the path is going to undefined. It should be OK to pass an empty array in this case?

Reply:
I think an empty array in the pre initialized phase is ok

Reply:
Got it. My suggestion here is that views() should return an empty array if the router is not initialized, rather than calling the view providers with incomplete information?